.github/workflows/build-wheels.yml

```diff
@@ -182,8 +181,16 @@ jobs:
           CIBW_BUILD_VERBOSITY: 2
           CIBW_MANYLINUX_X86_64_IMAGE: rustc-manylinux2014_x86_64
           CIBW_MANYLINUX_AARCH64_IMAGE: rustc-manylinux2014_aarch64
+          # METATENSOR_NO_LOCAL_DEPS is set to 1 when building a tag of
+          # metatensor-torch, which will force to use the version of
+          # metatensor-core already released on PyPI. Otherwise, this will use
+          # the version of metatensor-core from git checkout (in case there are
+          # unreleased breaking changes). This means we can not release breaking
+          # changes in metatensor-core and metatensor-torch by putting a tag on
+          # the same commit. Instead metatensor-core must be fully released
+          # before we start the build of metatensor-torch wheels.
```

Comment:
Does this answer your question @PicoCentauri ?

Reply:
Ahh yes. I should read more carefully. So if you don't set the env variable it will take the git version. This is also what I would expect and know from other packages.